_includes/apple-touch-icons.md

```diff
@@ -0,0 +1,9 @@
+{% comment -%}
+TODO: Figure out how to use front matter with proper arrays instead of split
+strings.
+{%- endcomment -%}
+{% assign design_guide_version_url = include.design_guide_version_url %}
+{%- assign sizes = "57, 60, 72, 76, 114, 120, 144, 152, 167, 180, 1024" | split: ',' -%}
```

Comment:
```suggestion
{%- assign sizes = "57", "60", "72", "76", "114", "120", "144", "152", "167", "180", "1024"-%}
```

Reply:
I don't think this is possible, unfortunately. At least I can't get it to work. 😕